src/lib.rs

```diff
@@ -510,6 +510,16 @@ pub struct WindowAttributes {
     /// The default is `false`.
     pub always_on_top: bool,
 
+    /// Whether the window will be centered on screen.
+    ///
+    /// The default is `true`.
+    pub center_window: bool,
+
+    /// Decides which monitor the window will start on. The user's primary monitor is `None`.
+    ///
+    /// The default is `None`,
+    pub start_monitor: Option<MonitorId>,
```

Comment:
I don't think it's unreasonable to require the user to select the monitor first (e.g. `EventsLoop::get_primary_monitor()`).

Why not remove `start_monitor` and only have:

```rust
center_window: Option<MonitorId>
```